renovate-config.json

```
@@ -0,0 +1,40 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "timezone": "Europe/Warsaw",
+  "extends": ["config:base", ":disableDependencyDashboard"],
+  "schedule": [
+    "before 5am on Monday"
+  ],
+  "packageRules": [
```

```suggestion
  "nvm": {
    "enabled": false
  },
  "ignoreDeps": [
    "node"
  ],
  "packageRules": [
```

My suggestion would be to disable Node upgrades, particularly if the `.nvmrc` file has been committed.

---

we're using `.node-version` in storefront-ui, but I understand your concern, thanks!